recipes/lightning-cloud/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "lightning-cloud" %}
+{% set version = "0.5.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lightning_cloud-{{ version }}.tar.gz
+  sha256: a5a138f4abbeffe66ee476fb9a8d621befac0434ffeeeec1cc00ccd3d72ffc09
+
+build:
+  entry_points:
+    - lightning_cloud=lightning_cloud.cli.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - click
+    - fastapi
+    - pyjwt
+    - python >=3.7
+    - requests
+    - rich
+    - six
+    - urllib3
+    - websocket-client
+
+test:
+  imports:
+    - lightning_cloud
+  commands:
+    - pip check
+    - lightning_cloud --help
+  requires:
+    - pip
+
+about:
+  home: https://lightning.ai
+  summary: Lightning Cloud
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - weiji14
```

Hi @Borda, would you like to be a conda-forge recipe maintainer for `lightning-cloud` (need a yes/no answer)? I noticed that you tried previously at https://github.com/conda-forge/staged-recipes/pull/19716 to add a recipe for this.

```suggestion
  recipe-maintainers:
    - Borda
    - weiji14
```

Let me know too if there's anyone from the Lightning team that might want to be on the recipe maintainer list.

---

cc: @lantiga